src/routes.go

```diff
@@ -0,0 +1,28 @@
+package api
+
+import "github.com/gin-gonic/gin"
+
+// HTTP methods.
+const (
+	GET    string = "GET"
+	HEAD   string = "HEAD"
+	POST   string = "POST"
+	PUT    string = "PUT"
+	DELETE string = "DELETE"
+	OPTION string = "OPTION"
+	PATCH  string = "PATCH"
+)
```

Comment:
Please use the consts in [http](https://pkg.go.dev/net/http#pkg-constants) package, and avoid redefining them ourselves. E.g. `http.MethodGet`.